server/manifest.yml

```diff
@@ -1,14 +1,18 @@
+declared-services:
+  nlc-email-phishing:
+    label: natural-language-classifier
+    plan: free
 applications:
 - path: .
   memory: 512M
   instances: 1
   domain: mybluemix.net
-  name: nlc-email-spam
-  host: nlc-email-spam
-
-  # disk_quota: 1024M
+  name: nlc-email-phishing
+  host: nlc-email-phishing
+  disk_quota: 1024M
+  random-route: true
   env:
```

Comment:
sure you want this set to true?

Reply:
no. This isn't working yet anyway. I'll just remove it from the patch set